components/pd_client/src/lib.rs

```diff
@@ -80,7 +80,9 @@ pub const INVALID_ID: u64 = 0;
 /// all the time.
 pub trait PdClient: Send + Sync {
     /// Returns the cluster ID.
-    fn get_cluster_id(&self) -> Result<u64>;
+    fn get_cluster_id(&self) -> Result<u64> {
```

Comment:
Is it a good idea to create a trait with all its method unimplemented? I think provided methods are methods that can work using required methods...

Reply:
emmm.. I just want to avoid duplication, because there is a mock pd client in `src/server/resolve.rs` with all methods unimplemented.